docs/doc1.md

```diff
@@ -0,0 +1,64 @@
+---
+id: doc1
```

Comment:
Is this name important?  It seems odd to use `doc1` since we'll have lots of them.